src/app/(Authentication)/Login/Password/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { router, useLocalSearchParams } from 'expo-router';
+import React, { useState } from 'react';
+import { Text, TextInput, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+import supabase from '../../../../supabase/createClient';
+import { signInUser } from '../../../../supabase/queries/auth';
+
+export default function LoginScreen() {
+  // const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [displayError, setDisplayError] = useState(false);
+  const { email } = useLocalSearchParams<{ email: string }>();
+
+  async function signInFunc(email: string, pword: string) {
```

Comment:
We can do this by updating the sign in query to return a boolean so that we can use it in this file to determine whether or not to display the error message